packages/api/tests/entities/Assignment.test.ts

```diff
@@ -1,16 +1,36 @@
 import { TestClient } from '../utils/TestClient';
+import { PathInput } from '../../types';
+import { ModuleType } from '../../src/Module/Module.entity';
 import * as random from '../../src/Database/seeders/random';
 
 beforeAll(async () => { await TestClient.start(); });
 afterAll(async () => { await TestClient.stop(); });
 
+export const assignment1 = "";
+
+export const moduleInput = {
+  name: 'module name',
+  icon: 'icon',
+  type: ModuleType.lesson,
+};
+
+export const pathInput: PathInput = {
+  name: 'Path name',
+  icon: 'icon',
+  description: 'Description text'
+};
+
 let moduleId: string;
+let path: Path;
 const setup = async () => {
   await TestClient.resetDatabase();
   await TestClient.workflowSignup();
-  const { id } = await TestClient.createPath();
-  const { id: modId } = await TestClient.createModule(random.moduleInput('name', id));
-  moduleId = modId;
+
+  path = await TestClient.createPath(pathInput);
+
+  const { id: modId } = await TestClient.createModule({ ...moduleInput, pathId: path.id });
+
+  assignment1 = random.assignmentInput(modId);
```

Comment:
Instead of creating the assingment1 here set moduleId = modId and you can use that below when you need to use random assignment